### PR TITLE
Added ipsiConnectionType() function

### PR DESCRIPTION
**Pull request by Rajdeepvirdi**
This is an External API use by the application to register with IPSI library and specify its role as "Caller"/"Server" along with a unique name

### CODE REVIEW

File: ipsi/src/ipsi.c [hunk 1]
```diff
@@ -185,11 +185,47 @@ static int initConnectionName(char *appName,char *connectionType){
...
 int ipsiConnectionType(char *appName, char *connectionType){
 
+	if(!(initConnectionName(appName,connectionType))){
+		IPSI_LOG("[IPSI] Invalid Connection Type, Exiting...\n");
+		return FAILURE;
+	}
+
+	//Initialize DBUS Code
+
+	//dbus_bus_get() will connect and register the client with DBus Daemon.
```
Comment (parthoCarleton) [inline ipsi/src/ipsi.c:200]: can you please remove this Comment

File: ipsi/src/ipsi.c [hunk 2]
```diff
@@ -185,11 +185,47 @@ static int initConnectionName(char *appName,char *connectionType){
...
+		IPSI_LOG("[IPSI] Connection Error (%s)\n", err.message);
+		dbus_error_free(&err);
+		return FAILURE;
+	}
+	if (NULL == conn) {
+		IPSI_LOG("[IPSI] Connection Null\n");
+		return FAILURE;
+	}
+
+	//dbus_bus_request_name() asks the DBus to assign the given name (connectionName) to the connection (conn).
```
Comment (parthoCarleton) [inline ipsi/src/ipsi.c:213]: Remove this and any excessive comment from this function

**Review by parthoCarleton — CHANGES_REQUESTED** [review]

File: ipsi/src/ipsi.c (hunk 1 above)
Comment (Rajdeepvirdi) [reply ipsi/src/ipsi.c:200]: sure!

File: ipsi/src/ipsi.c (hunk 2 above)
Comment (Rajdeepvirdi) [reply ipsi/src/ipsi.c:213]: okay.

**Review by parthoCarleton — APPROVED** [review]

**Review by ashwini1025 — APPROVED** [review]
Look Good.

**Pull request merged**